src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -26,6 +26,11 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
 
+        <service id="serializer.normalizer.datetime" class="Symfony\Component\Serializer\Normalizer\DateTimeNormalizer" public="false">
+            <!-- Run before serializer.normalizer.object -->
+            <tag name="serializer.normalizer" priority="-910" />
```

Comment:
I would need to change the priority in #17603 because you would want this to run first